test/extended/authorization/per_node_validatingadmissionpolicy.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingAdmissionPolicy
+metadata:
+  name: "only-allow-name-matching-node-configmaps"
+spec:
+  failurePolicy: Fail
+  matchConstraints:
+    resourceRules:
+      - apiGroups:   [""]
+        apiVersions: ["v1"]
+        # notice you cannot restrict DELETEs.  Your per-node actor should not have deletion powers.
```

Comment:
this detail on delete is worth being sure is clear.

Reply:
This is because we don't have the body of the object right?

Wouldn't it be safer to include logic to fail all delete attempts?

Reply:
> This is because we don't have the body of the object right?

Yes



> Wouldn't it be safer to include logic to fail all delete attempts?

I think it would be safer to not grant deletion powers in RBAC.